number_parser/parser.py

```diff
@@ -131,14 +133,30 @@ def _build_number(token_list, lang_data):
     return value_list
 
 
-def _tokenize(input_string):
+def _tokenize(input_string, language):
     """Breaks string on any non-word character."""
+    input_string = input_string.replace('\xad', '')
+    if language in RE_BUG_LANGUAGES:
+        return input_string.split()
     tokens = re.split(r'(\W)', input_string)
     return tokens
 
 
+def _strip_accents(word):
+    """Removes accent from the input word."""
+    return ''.join(char for char in unicodedata.normalize('NFD', word) if unicodedata.category(char) != 'Mn')
+
+
 def _normalize_tokens(token_list):
-    return [token.lower() for token in token_list]
+    """Converts all tokens to lowercase then removes accents."""
+    lower_case_tokens = [token.lower() for token in token_list]
+    normalized_tokens = [_strip_accents(token) for token in lower_case_tokens]
+    return normalized_tokens
+
+
+def _normalise_dict(lang_dict):
```

Comment:
Good job with this function. I would rename it to `_normalize_dict` (with `z`) to be more consistent.